cmd/natc/natc.go

```diff
@@ -139,6 +152,10 @@ type connector struct {
 	v6ULA netip.Prefix
 
 	perPeerMap syncs.Map[tailcfg.NodeID, *perPeerState]
+
+	// ignoreDsts is initialized at start up with the contents of --ignore-destinations (if none it is nil)
+	// it is never mutated, only used to check destinations IP addresses against.
+	ignoreDsts *bart.Table[bool]
```

Comment:
This also needs an explanation for the intent. A brief explanation of how its use would help. Like what behavior changes when an IP is in this set